samples/bluetooth/connection_event_trigger/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _ble_connection_event_trigger:
+
+Bluetooth: Event Trigger
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Event Trigger sample showcases the Event Trigger feature, a proprietary
+extension of the Bluetooth® protocol which can be used to schedule sampling, GPIO toggling, or other SW- or HW-based
+user activities in synchronization with individual BLE role instances, such as a central or peripheral link.
+
+Requirements
+************
+
+The sample supports the following development kits:
```

Comment:
Could you add a note about how it works and why it can't work on 53-app? I.e. the requirement for the PPI triggered from the LL.

Also could it work on the app core? I thought we had a PPI bridge. Or I'm confusing with 54?

Reply:
To use this on the app core on 53 I imagine you would need to implement some kind of IPC, each core has its own DPPI controller and I don't know of any hardware-based way to somehow link them

Reply:
There is https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf5340%2Fipc.html&cp=4_0_0_6_15_0&anchor=unique_459239316, which seems similar to the IPCTs on 54H, but I have never used them.